node-api/src/metadata.rs

```diff
@@ -17,10 +17,8 @@ use frame_metadata::{
 	StorageEntryMetadata, META_RESERVED,
 };
 use scale_info::{form::PortableForm, PortableRegistry, Type};
-use sp_core::storage::StorageKey;
-
-#[cfg(feature = "std")]
 use serde::Serialize;
```

Comment:
Serde is no_std compatible, so no need to hide behind a std feature only flag.. or was there another reason @echevrier ?

Reply:
Sorry my bad. Seems like serde_derive is necessary, simply using `Serialize` is not enough. You can ignore that one.